chat-components/database/src/main/java/com/pubnub/components/repository/message/DefaultMessageRepository.kt

```diff
@@ -20,6 +21,7 @@ import kotlinx.coroutines.flow.firstOrNull
  * @param messageDao MessageDao implementation
  */
 class DefaultMessageRepository(
+    private val database: RoomDatabase,
```

Comment:
@kazydek Small docs update will be needed here, both `DefaultMessageRepository` and DefaultMessageActionRepository` are implementing the `runInTransaction` method and have an extra `database` parameter (so `MessageRepository` and `MessageActionRepository` interfaces have `fun runInTransaction(body: () -> Unit)`)

Reply:
We're merging to master, will fix that before release